src/main/java/budgetbuddy/logic/commands/CommandCategory.java

```diff
@@ -3,6 +3,19 @@
 /**
  * Represents the category that a command belongs to.
  */
+
 public enum CommandCategory {
-    RULE, ACCOUNT, TRANSACTION, LOAN, LOAN_SPLIT, SCRIPT;
+    RULE, ACCOUNT, TRANSACTION, LOAN, SCRIPT,
+    LOAN_SPLIT;
+
+    public boolean isSecondaryCategory() {
+        return this == LOAN_SPLIT;
+    }
+
+    public CommandCategory getPrimaryCategory(CommandCategory secondary) {
+        if (secondary == LOAN_SPLIT) {
+            return LOAN;
+        }
+        return this;
+    }
```

Comment:
Wow hmm this seems a little bit hardcoded ish

Reply:
I agree, but figured that:

- If someone else wants to add another secondary category, they can add on to the checks in these methods.

- No one else is likely to use it anyway.

The main problem I had to solve was to signal to the UI that a category was not going to the match the tab name. I’m open to suggestions on how better to accomplish this, although this seems like the easiest way.

Reply:
fair enough!